gateway/1.5/modules/ROOT/pages/flex-gateway-renew-certificate.adoc

```diff
@@ -0,0 +1,340 @@
+
+[tabs]
+====
+Docker::
++
+[source,ssh]
+----
+docker run --entrypoint flexctl mulesoft/flex-gateway registration inspect --help
+----
+
+Podman::
++
+[source,ssh]
+----
+podman run --entrypoint flexctl docker.io/mulesoft/flex-gateway registration inspect --help
+----
+====
+
+Refer to the following topics for more information about inspection command usage:
+
+* <<inspect-with-custom-url,Inspect Registration Status with an Admin API URL>>
+* <<inspect-with-reg-file,Inspect Registration Status with a Registration File>>
+
+[NOTE]
+====
+If you call `flexctl registration inspect` without flags, the command inspects registration status by using the default Admin API URL.
+====
+
+[[inspect-with-custom-url]]
+=== Inspect Registration Status with an Admin API URL
+
+You can inspect a registration by specifying the base URL of the Flex Gateway Admin API. The default value is `http://localhost:9999`. Omit the `--flex-gateway-admin-api-url` flag if using the default value.
+
+[source,ssh]
+----
+flexctl registration inspect \
+--flex-gateway-admin-api-url=<your-custom-admin-api-url>
+----
+
+For Flex Gateway running in a container, invoke `inspect` as part of the container `exec` command. Replace `<container-name-or-id>` with the running container reference:
+
+[tabs]
+====
+Docker::
++
+[source,ssh]
+----
+docker exec -u $UID <container-name-or-id> flexctl registration inspect \
+--flex-gateway-admin-api-url=<your-custom-admin-api-url>
+----
+
+Podman::
++
+[source,ssh]
+----
+podman exec -u $UID <container-name-or-id> flexctl registration inspect \
+--flex-gateway-admin-api-url=<your-custom-admin-api-url>
+----
+====
+
+Invoking the command results in the following example output:
+
+[source,text]
+----
+{“expiration_date”: “2024-01-16 16:20:30 +0000 UTC”}
+----
+
+[[inspect-with-reg-file]]
+=== Inspect Registration Status with a Registration File
+
+You can inspect a registration by extracting the information from your `registration.yaml` file. Specify the path and filename.
+
+[source,ssh]
+----
+flexctl registration inspect --file=<path-to-registration-file>
+----
+
+For Flex Gateway running in a container, invoke `inspect` as part of the container `run` command:
+
+[tabs]
+====
+Docker::
++
+[source,ssh]
+----
+docker run --entrypoint flexctl \
+-v <path-to-registration-directory>:/registration \
+-u $UID mulesoft/flex-gateway \
+registration inspect --file=/registration/<registration-filename>
+----
+
+Podman::
++
+[source,ssh]
+----
+podman run --entrypoint flexctl --userns=keep-id \
+-v <path-to-registration-directory>:/registration:Z \
+-u $UID docker.io/mulesoft/flex-gateway \
+registration inspect --file=/registration/<registration-filename>
+----
+====
+
+Invoking the command results in the following example output:
+
+[source,text]
+----
+{“expiration_date”: “2024-01-16 16:20:30 +0000 UTC”}
+----
+
+== See Also
+
+* xref:flex-install.adoc[Download Flex Gateway]
```

Comment:
If you omit the the hardcoded text, it will propagate automatically and be easier to maintain.

Reply:
Fixed!